pkg/hcl_interpreter/term.go

```diff
@@ -297,13 +302,18 @@ func (t Term) Evaluate(
 		diagnostics = append(diagnostics, diags...)
 
 		evalWithEach := func(key cty.Value, value cty.Value) cty.Value {
-			val, diags := t.evaluateExpr(func(e hcl.Expression, v cty.Value) (cty.Value, hcl.Diagnostics) {
-				v = MergeVal(v, NestVal(LocalName{t.iterator}, cty.ObjectVal(map[string]cty.Value{
-					"key":   key,
-					"value": value,
-				})))
-				return evalExpr(e, v)
-			})
+			val, diags := t.evaluateExpr(
```

Comment:
It doesn't seem like there's actual changes here -- maybe different settings on `gofmt` or a linter?  I don't mind having this either way, just curious.

Reply:
i guess different settings for gofmt? I'd rather not have code changes just for the sake of it, so I'll try to fix it